apps/smart-contracts/token/test/fixtures/PPOFixtures.ts

```diff
@@ -1,7 +1,11 @@
 import { ethers, upgrades } from 'hardhat'
 import { PPO } from '../../types/generated'
 
-export async function ppoFixture(governanceAddress: string): Promise<PPO> {
+export async function ppoFixture(
+  governanceAddress: string,
```

Comment:
nit:
```suggestion
  nominatedOwnerAddress: string,
```
for consistency with actual function